### DISCUSSION TITLE
Trigger github action from googlesheet

### DISCUSSION THREAD

**Discussion by WIssamsaliba** [community/community#34717] [Apps, API and Webhooks]
Hey, I am creating a python code that helps me fill out forms depending on values inside a google sheet. But my main problem is how can I trigger a workflow from github when I press a button from google sheet.(I want to run a code on github cloud when i press a button on googlesheet.) Is this possible? and If yes may someone explain how can I do it or an article that helps me understand how to write my code down.

**Comment by airtower-luna** [comment]
You can create `workflow_dispatch` events [via the REST API](https://docs.github.com/en/rest/actions/workflows#create-a-workflow-dispatch-event). I don't know whether you can integrate that into your sheet, mind that the request requires authentication.
